evergreen.yml

```diff
@@ -177,6 +177,22 @@ functions:
           ${prepare_shell}
           cargo fmt --all --  --check
 
+  "check unused dependencies":
+    - command: shell.exec
+      type: test
+      params:
+        shell: bash
+        working_dir: mongosql-odbc-driver
+        script: |
+          ${prepare_shell}
+          cargo install cargo-machete
+          cargo machete
+          RETURN=$?
+          if [ $RETURN -ne 0 ]; then
+            echo "Unused dependencies found"
```

Comment:
[nit] Can you use >&2 echo "xxx" to send it to stderr instead of stdout so that it shows in red on Evergreen.